### PR TITLE
jwt.de.md german translation for #977

### PR DESCRIPTION
**Pull request by plse73**

### CODE REVIEW

**Comment by plse73** [comment]
This is my first contribution to the Git community and the processing of open issues. Since I am German and my native language level is German, I hoped to be able to offer added value here. Hopefully I could help and of course I'm looking forward to your feedback. 
Best regards

**Comment by plse73** [comment]
Here I have made the translation into German. Unfortunately I could not find the desired label(translation-update) to add it to the request. Clarification would be great here so that I can take it into account in the future.

**Comment by 0xTim** [comment]
Thanks @plse73 ! I've added the label, it might be you need to scroll to see it or not have permissions yet. I'll leave this for a bit to see if anyone wants to proof read!

File: docs/security/jwt.de.md [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+        ]),
+        // Weitere Ziele...
+    ]
+)
+```
+
+Wenn du das Manifest direkt in Xcode bearbeitest, werden die Änderungen automatisch übernommen und die neue Abhängigkeit wird beim Speichern der Datei abgerufen. Andernfalls führe `swift package resolve` aus, um die neue Abhängigkeit abzurufen.
+### Konfiguration
+
+Das JWT-Modul fügt `Application` eine neue Eigenschaft `jwt hinzu, die für die Konfiguration verwendet wird. Um JWTs zu signieren oder zu verifizieren, musst du einen Schlüssel hinzufügen. Der einfachste Signieralgorithmus ist HS256 oder HMAC mit SHA-256.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:29]:
```suggestion
Das JWT-Modul fügt `Application` eine neue Eigenschaft `jwt` hinzu, die für die Konfiguration verwendet wird. Um JWTs zu signieren oder zu verifizieren, musst du einen Schlüssel hinzufügen. Der einfachste Signieralgorithmus ist HS256 oder HMAC mit SHA-256.
```

File: docs/security/jwt.de.md [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+
+// HMAC mit SHA-256 Unterzeichner hinzufügen.
+await app.jwt.keys.addHMAC(key: "secret", digestAlgorithm: .sha256)
+```
+
+!!! Hinweis: 
+Das `await` Schlüsselwort ist erforderlich, da die Schlüsselsammlung ein `actor` ist.
+
+
+Der `HS256` Unterzeichnern benötigt einen Schlüssel zur Initialisierung. Im Gegensatz zu anderen Unterzeichnern wird dieser eine Schlüssel sowohl zum Signieren _als auch_ zum Verifizieren von Tokens verwendet. Erfahre mehr über die verfügbaren [Algorithmen](#Algorithmen) unten.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:42]:
```suggestion
Der `HS256` Unterzeichner benötigt einen Schlüssel zur Initialisierung. Im Gegensatz zu anderen Unterzeichnern wird dieser eine Schlüssel sowohl zum Signieren _als auch_ zum Verifizieren von Tokens verwendet. Erfahre mehr über die verfügbaren [Algorithmen](#Algorithmen) unten.
```

File: docs/security/jwt.de.md [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+### Payload
+
+Versuchen wir, das folgende JWT-Beispiel zu überprüfen.
+
+```swift
+eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ2YXBvciIsImV4cCI6NjQwOTIyMTEyMDAsImFkbWluIjp0cnVlfQ.lS5lpwfRNSZDvpGQk6x5JI1g40gkYCOWqbc3J_ghowo
+```
+Sie können den Inhalt dieses Tokens überprüfen, indem Sie [jwt.io](https://jwt.io) besuchen und das Token in den Debugger einfügen. Setzen Sie den Schlüssel im Abschnitt "Verify Signature" auf `secret`.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:50]:
```suggestion
Du kannst den Inhalt dieses Tokens überprüfen, indem du [jwt.io](https://jwt.io) besuchst und das Token in den Debugger einfügst. Setz den Schlüssel im Abschnitt "Verify Signature" auf `secret`.
```

File: docs/security/jwt.de.md [hunk 4]
```diff
@@ -0,0 +1,411 @@
...
+
+```json
+{
+   "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ2YXBvciIsImV4cCI6NjQwOTIyMTEyMDAsImFkbWluIjp0cnVlfQ.lS5lpwfRNSZDvpGQk6x5JI1g40gkYCOWqbc3J_ghowo"
+}
+```
+
+## Authentifizierung
+
+Weitere Informationen zur Verwendung von JWT mit Vapors Authentifizierungs-API finden Sie unter [Authentifizierung &rarr; JWT](authentication.md#jwt).
```
Comment (ptoffy) [inline docs/security/jwt.de.md:155]:
```suggestion
Weitere Informationen zur Verwendung von JWT mit Vapors Authentifizierungs-API findest du unter [Authentifizierung &rarr; JWT](authentication.md#jwt).
```

File: docs/security/jwt.de.md [hunk 5]
```diff
@@ -0,0 +1,411 @@
...
+await app.jwt.keys.addHMAC(key: "secret", digestAlgorithm: .sha256)
+```
+
+### RSA
+
+RSA ist der am häufigsten verwendete JWT-Signieralgorithmus. Er unterstützt unterschiedliche öffentliche und private Schlüssel. Das bedeutet, dass ein öffentlicher Schlüssel verteilt werden kann, um die Authentizität der JWTs zu überprüfen, während der private Schlüssel, der sie erzeugt, geheim gehalten wird.
+
+!!! Warnung
+
+Das JWT-Paket von Vapor unterstützt keine RSA-Schlüssel mit einer Größe von weniger als 2048 Bit. Da RSA aus Sicherheitsgründen von der NIST nicht mehr empfohlen wird, sind RSA-Schlüssel hinter einem `Insecure`-Namensraum eingeschlossen, um von ihrer Verwendung abzuschrecken.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:180]:
```suggestion
!!! warning "Warnung"
    Das JWT-Paket von Vapor unterstützt keine RSA-Schlüssel mit einer Größe von weniger als 2048 Bit. Da RSA aus Sicherheitsgründen von der NIST nicht mehr empfohlen wird, sind RSA-Schlüssel hinter einem `Insecure`-Namensraum eingeschlossen, um von ihrer Verwendung abzuschrecken.
```

File: docs/security/jwt.de.md [hunk 6]
```diff
@@ -0,0 +1,411 @@
...
+let key = try Insecure.RSA.PrivateKey(
+    modulus: modulus, 
+    exponent: publicExponent, 
+    privateExponent: privateExponent
+)
+```
+Der Initialisierer für den öffentlichen Schlüssel ist ähnlich.
+
+```swift
+// Initialisieren Sie einen öffentlichen RSA-Schlüssel mit Komponenten.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:194]:
```suggestion
// Initialisiere einen öffentlichen RSA-Schlüssel mit Komponenten.
```

File: docs/security/jwt.de.md [hunk 7]
```diff
@@ -0,0 +1,411 @@
...
+
+RSA ist der am häufigsten verwendete JWT-Signieralgorithmus. Er unterstützt unterschiedliche öffentliche und private Schlüssel. Das bedeutet, dass ein öffentlicher Schlüssel verteilt werden kann, um die Authentizität der JWTs zu überprüfen, während der private Schlüssel, der sie erzeugt, geheim gehalten wird.
+
+!!! Warnung
+
+Das JWT-Paket von Vapor unterstützt keine RSA-Schlüssel mit einer Größe von weniger als 2048 Bit. Da RSA aus Sicherheitsgründen von der NIST nicht mehr empfohlen wird, sind RSA-Schlüssel hinter einem `Insecure`-Namensraum eingeschlossen, um von ihrer Verwendung abzuschrecken.
+
+Um einen RSA-Signierer zu erstellen, muss zunächst ein `RSAKey` initialisiert werden. Dies kann durch Übergabe der Komponenten geschehen.
+```swift
+// Initialisieren Sie einen privaten RSA-Schlüssel mit Komponenten.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:184]:
```suggestion
// Initialisiere einen privaten RSA-Schlüssel mit Komponenten.
```

File: docs/security/jwt.de.md [hunk 8]
```diff
@@ -0,0 +1,411 @@
...
+
+```swift
+// Initialisieren Sie einen öffentlichen RSA-Schlüssel mit Komponenten.
+let key = try Insecure.RSA.PublicKey(
+    modulus: modulus, 
+    exponent: publicExponent
+)
+```
+
+Sie können auch eine PEM-Datei laden:
```
Comment (ptoffy) [inline docs/security/jwt.de.md:201]:
```suggestion
Du kannst auch eine PEM-Datei laden:
```

File: docs/security/jwt.de.md [hunk 9]
```diff
@@ -0,0 +1,411 @@
...
+7FPo5Cq8FkvrdDzeacwRSxYuIq1LtYnd6I30qNaNthntjvbqyMmBulJ1mzLI+Xg/
+aX4rbSL49Z3dAQn8vQIDAQAB
+-----END PUBLIC KEY-----
+"""
+
+// Initialisieren Sie einen RSA-Schlüssel mit public pem.
+let key = try Insecure.RSA.PublicKey(pem: rsaPublicKey)
+```
+
+Verwenden Sie `Insecure.RSA.PrivateKey`, um private RSA-PEM-Schlüssel zu laden. Diese beginnen mit:
```
Comment (ptoffy) [inline docs/security/jwt.de.md:217]:
```suggestion
Verwende `Insecure.RSA.PrivateKey`, um private RSA-PEM-Schlüssel zu laden. Diese beginnen mit:
```

File: docs/security/jwt.de.md [hunk 10]
```diff
@@ -0,0 +1,411 @@
...
+```
+-----BEGIN RSA PRIVATE KEY-----
+```
+
+Sobald Sie den RSA-Schlüssel haben, können Sie ihn mit der Methode `addRSA` hinzufügen.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:222]:
```suggestion
Sobald du den RSA-Schlüssel hast, kannst du ihn mit der Methode `addRSA` hinzufügen.
```

Comment (ptoffy) [inline docs/security/jwt.de.md:222]:
```suggestion
Sobald du den RSA-Schlüssel hast, kannst du ihn mit der Methode `addRSA` hinzufügen.
```

File: docs/security/jwt.de.md [hunk 11]
```diff
@@ -0,0 +1,411 @@
...
+let key = Insecure.RSA.PublicKey(pem: publicKey)
+try app.jwt.keys.addPSS(key: key, digestAlgorithm: .sha256)
+```
+
+### ECDSA
+
+ECDSA ist ein modernerer Algorithmus, der mit RSA vergleichbar ist. Er gilt bei einer bestimmten Schlüssellänge als sicherer als RSA[^1]. Sie sollten jedoch eigene Nachforschungen anstellen, bevor Sie sich entscheiden.
+[^1]: https://www.ssl.com/article/comparing-ecdsa-vs-rsa/
+
+Wie bei RSA können Sie ECDSA-Schlüssel über PEM-Dateien laden:
```
Comment (ptoffy) [inline docs/security/jwt.de.md:248]:
```suggestion
Wie bei RSA kannst du ECDSA-Schlüssel über PEM-Dateien laden:
```

File: docs/security/jwt.de.md [hunk 12]
```diff
@@ -0,0 +1,411 @@
...
+
+```swift
+let ecdsaPublicKey = """
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE2adMrdG7aUfZH57aeKFFM01dPnkx
+C18ScRb4Z6poMBgJtYlVtd9ly63URv57ZW0Ncs1LiZB7WATb3svu+1c7HQ==
+-----END PUBLIC KEY-----
+"""
+
+// Initialisieren Sie einen ECDSA-Schlüssel mit öffentlichem PEM.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:258]:
```suggestion
// Initialisiere einen ECDSA-Schlüssel mit öffentlichem PEM.
```

File: docs/security/jwt.de.md [hunk 13]
```diff
@@ -0,0 +1,411 @@
...
+
+Es gibt drei ECDSA-Algorithmen, je nach verwendeter Kurve:
+- `ES256`: ECDSA mit einer P-256 Kurve und SHA-256
+- `ES384`: ECDSA mit einer P-384 Kurve und SHA-384
+- `ES512`: ECDSA mit einer P-521 Kurve und SHA-512
+
+Alle Algorithmen bieten sowohl einen öffentlichen Schlüssel als auch einen privaten Schlüssel,
+wie z. B. `ES256PublicKey` und `ES256PrivateKey`.
+
+Sie können auch einen zufälligen ECDSA-Schlüssel erzeugen, indem Sie den leeren Initialisierer verwenden. Dies ist für Tests nützlich.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:270]:
```suggestion
Du kannst auch einen zufälligen ECDSA-Schlüssel erzeugen, indem du den leeren Initialisierer verwendest. Dies ist für Tests nützlich.
```

File: docs/security/jwt.de.md [hunk 14]
```diff
@@ -0,0 +1,411 @@
...
+
+```swift
+let key = ES256PrivateKey()
+```
+
+Sobald Sie den ECDSAKey haben, können Sie ihn mit der Methode `addECDSA` zur Schlüsselsammlung hinzufügen.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:276]:
```suggestion
Sobald du den ECDSAKey hast, kannst du ihn mit der Methode `addECDSA` zur Schlüsselsammlung hinzufügen.
```

File: docs/security/jwt.de.md [hunk 15]
```diff
@@ -0,0 +1,411 @@
...
+
+```swift
+// ECDSA mit SHA-256 Unterzeichner hinzufügen.
+try await app.jwt.keys.addECDSA(key: ES256PublicKey(pem: ecdsaPublicKey))
+```
+
+### Schlüsselidentifikator (kid)
+
+Wenn Sie mehrere Algorithmen verwenden, können Sie Schlüsselbezeichner (`kid`) verwenden, um sie zu unterscheiden. Wenn Sie einen Algorithmus konfigurieren, übergeben Sie den Parameter "kid".
```
Comment (ptoffy) [inline docs/security/jwt.de.md:285]:
```suggestion
Wenn du mehrere Algorithmen verwendest, kannst du Schlüsselbezeichner (`kid`) verwenden, um sie zu unterscheiden. Wenn du einen Algorithmus konfigurierst, übergebe den `kid` Parameter.
```

File: docs/security/jwt.de.md [hunk 16]
```diff
@@ -0,0 +1,411 @@
...
+
+```swift
+// HMAC mit SHA-256 Unterzeichner namens "a" hinzufügen.
+await app.jwt.keys.addHMAC(key: "foo", digestAlgorithm: .sha256, kid: "a")
+// HMAC mit SHA-256 Unterzeichner namens "b" hinzufügen.
+await app.jwt.keys.addHMAC(key: "bar", digestAlgorithm: .sha256, kid: "b")
+```
+
+Wenn Sie JWTs signieren, übergeben Sie den Parameter `kid` für den gewünschten Unterzeichner.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:294]:
```suggestion
Wenn du JWTs signierst, übergebe den `kid` Parameter für den gewünschten Unterzeichner.
```

File: docs/security/jwt.de.md [hunk 17]
```diff
@@ -0,0 +1,411 @@
...
+```
+Dadurch wird der Name des Unterzeichners automatisch in das Feld `"kid"` des JWT-Headers aufgenommen. Bei der Verifizierung des JWT wird dieses Feld verwendet, um den entsprechenden Unterzeichner zu finden.
+
+```swift
+// Überprüfung mit dem im "kid"-Kopf angegebenen Unterzeichner.
+// Wenn kein "kid"-Header vorhanden ist, wird der Standardunterzeichner verwendet.
+let payload = try await req.jwt.verify(as: TestPayload.self)
+```
+
+Da [JWKs](#jwk) bereits `kid` Werte enthalten, müssen Sie diese bei der Konfiguration nicht angeben.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:308]:
```suggestion
Da [JWKs](#jwk) bereits `kid` Werte enthalten, musst du diese bei der Konfiguration nicht angeben.
```

File: docs/security/jwt.de.md [hunk 18]
```diff
@@ -0,0 +1,411 @@
...
+|`aud`|`AudienceClaim`|`verifyIntendedAudience(includes:)`|
+|`exp`|`ExpirationClaim`|`verifyNotExpired(currentDate:)`|
+|`jti`|`IDClaim`|n/a|
+|`iat`|`IssuedAtClaim`|n/a|
+|`iss`|`IssuerClaim`|n/a|
+|`locale`|`LocaleClaim`|n/a|
+|`nbf`|`NotBeforeClaim`|`verifyNotBefore(currentDate:)`|
+|`sub`|`SubjectClaim`|n/a|
+
+Alle Ansprüche sollten mit der Methode `JWTPayload.verify` überprüft werden. Wenn der Anspruch eine spezielle verify-Methode hat, können Sie diese verwenden. Andernfalls greifen Sie mit `value` auf den Wert des Anspruchs zu und prüfen, ob er gültig ist.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:331]:
```suggestion
Alle Ansprüche sollten mit der Methode `JWTPayload.verify` überprüft werden. Wenn der Anspruch eine spezielle verify-Methode hat, kannst du diese verwenden. Andernfalls greife mit `value` auf den Wert des Anspruchs zu und prüfen, ob er gültig ist.
```

File: docs/security/jwt.de.md [hunk 19]
```diff
@@ -0,0 +1,411 @@
...
+Ein JSON Web Key (JWK) ist eine JavaScript Object Notation (JSON) Datenstruktur, die einen kryptografischen Schlüssel darstellt ([RFC7517](https://tools.ietf.org/html/rfc7517)). Diese werden üblicherweise verwendet, um Clients mit Schlüsseln zur Verifizierung von JWTs zu versorgen.
+
+Zum Beispiel hostet Apple sein _Sign in with Apple_ JWKS unter der folgenden URL.
+
+```http
+GET https://appleid.apple.com/auth/keys
+```
+
+Sie können dieses JSON Web Key Set (JWKS) zu Ihren `JWTSigners` hinzufügen.
+Sie können dann JWTs von Apple an die Methode `verify` übergeben. Der Schlüsselbezeichner (`kid`) im JWT-Header wird verwendet, um automatisch den richtigen Schlüssel für die Verifizierung auszuwählen.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:344]: This needs updating in the English version too, although we're about to update the docs anyway with v5 coming along so not to worry
```suggestion
Du kannst dieses JSON Web Key Set (JWKS) zu `JWTKeyCollection` hinzufügen.
Du kannst dann JWTs von Apple an die Methode `verify` übergeben. Der Schlüsselbezeichner (`kid`) im JWT-Header wird verwendet, um automatisch den richtigen Schlüssel für die Verifizierung auszuwählen.
```

File: docs/security/jwt.de.md [hunk 20]
```diff
@@ -0,0 +1,411 @@
...
+
+JWT-Herausgeber können ihre JWKS rotieren, was bedeutet, dass Sie gelegentlich einen neuen Download durchführen müssen. Siehe Vapor's unterstützte JWT [Vendors](#vendors) Liste unten für APIs, die dies automatisch tun.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:346]:
```suggestion
JWT-Herausgeber können ihre JWKS rotieren, was bedeutet, dass du gelegentlich einen neuen Download durchführen müsst. Siehe Vapor's unterstützte JWT [Vendors](#vendors) Liste unten für APIs, die dies automatisch tun.
```

File: docs/security/jwt.de.md [hunk 21]
```diff
@@ -0,0 +1,411 @@
...
+
+## Vendors
+
+Vapor bietet APIs für die Verarbeitung von JWTs von den unten aufgeführten populären Emittenten.
+
+### Apple
+
+Konfigurieren Sie zunächst die Kennung Ihrer Apple-Anwendung.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:354]:
```suggestion
Konfiguriere zunächst die Kennung deiner Apple-Anwendung.
```

File: docs/security/jwt.de.md [hunk 22]
```diff
@@ -0,0 +1,411 @@
...
+
+```swift
+// Konfigurieren Sie die Apple-App-Kennung.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:357]:
```suggestion
// Konfiguriere die Apple-App-Kennung.
```

File: docs/security/jwt.de.md [hunk 23]
```diff
@@ -0,0 +1,411 @@
...
+app.jwt.apple.applicationIdentifier = "..."
+```
+
+Verwenden Sie dann die Hilfsfunktion `req.jwt.apple`, um ein Apple JWT abzurufen und zu überprüfen.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:361]:
```suggestion
Verwende dann die Hilfsfunktion `req.jwt.apple`, um ein Apple JWT abzurufen und zu überprüfen.
```

File: docs/security/jwt.de.md [hunk 24]
```diff
@@ -0,0 +1,411 @@
...
+app.get("apple") { req async throws -> HTTPStatus in
+    let token = try await req.jwt.apple.verify()
+    print(token) // AppleIdentityToken
+    return .ok
+}
+```
+
+### Google
+
+Konfigurieren Sie zunächst Ihre Google-Anwendungskennung und den G Suite-Domänennamen.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:374]:
```suggestion
Konfiguriere zunächst Ihre Google-Anwendungskennung und den G Suite-Domänennamen.
```

File: docs/security/jwt.de.md [hunk 25]
```diff
@@ -0,0 +1,411 @@
...
+
+```swift
+// Konfigurieren Sie den Bezeichner der Google-App und den Domänennamen.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:377]:
```suggestion
// Konfiguriere den Bezeichner der Google-App und den Domänennamen.
```

File: docs/security/jwt.de.md [hunk 26]
```diff
@@ -0,0 +1,411 @@
...
+app.jwt.google.applicationIdentifier = "..."
+app.jwt.google.gSuiteDomainName = "..."
+```
+
+Verwenden Sie dann die Hilfe `req.jwt.google`, um ein Google JWT zu holen und zu überprüfen.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:382]: As above

File: docs/security/jwt.de.md [hunk 27]
```diff
@@ -0,0 +1,411 @@
...
+app.get("google") { req async throws -> HTTPStatus in
+    let token = try await req.jwt.google.verify()
+    print(token) // GoogleIdentityToken
+    return .ok
+}
+```
+
+### Microsoft
+
+Konfigurieren Sie zunächst die Kennung Ihrer Microsoft-Anwendung.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:395]: As above

File: docs/security/jwt.de.md [hunk 28]
```diff
@@ -0,0 +1,411 @@
...
+
+```swift
+// Konfigurieren Sie die Kennung der Microsoft-App.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:398]: As above

File: docs/security/jwt.de.md [hunk 29]
```diff
@@ -0,0 +1,411 @@
...
+app.jwt.microsoft.applicationIdentifier = "..."
+```
+
+Verwenden Sie dann die Hilfsfunktion `req.jwt.microsoft`, um ein Microsoft JWT abzurufen und zu überprüfen.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:402]: As above

File: docs/security/jwt.de.md [hunk 30]
```diff
@@ -0,0 +1,411 @@
...
+
+```swift
+import JWT
+
+// HMAC mit SHA-256 Unterzeichner hinzufügen.
+await app.jwt.keys.addHMAC(key: "secret", digestAlgorithm: .sha256)
+```
+
+!!! Hinweis: 
+Das `await` Schlüsselwort ist erforderlich, da die Schlüsselsammlung ein `actor` ist.
```
Comment (ptoffy) [inline docs/security/jwt.de.md:39]:
```suggestion
!!! tip "Hinweis": 
    Das `await` Schlüsselwort ist erforderlich, da die Schlüsselsammlung ein `actor` ist.
```

**Review by ptoffy — CHANGES_REQUESTED** [review]
Thanks for this! I have a few nits about using a consistent Anrede in the whole file. Also let's make sure the spacing around titles etc is consistent (e.g leave a space below (sub)titles). Looks good otherwise

**Review by plse73 — COMMENTED** [review]
Thank you very much for your detailed feedback and comments regarding my mistakes. I hope that I have made the correction in a way that it is now visible. I had a note regarding one of the corrections.

Best regards,

Philip

**Review by ptoffy — APPROVED** [review]

**Comment by plse73** [comment]
@ptoffy do I have to do anything else, because the merging is blocked for me. I am not an authorized user for this branch. How do I proceed?

**Pull request merged**